paychmgr/store.go

```diff
@@ -376,14 +379,29 @@ func (ps *Store) GetMessage(ctx context.Context, mcid cid.Cid) (*MsgInfo, error)
 
 // OutboundActiveByFromTo looks for outbound channels that have not been
 // settled, with the given from / to addresses
-func (ps *Store) OutboundActiveByFromTo(ctx context.Context, from address.Address, to address.Address) (*ChannelInfo, error) {
+func (ps *Store) OutboundActiveByFromTo(ctx context.Context, sma stateManagerAPI, from address.Address, to address.Address) (*ChannelInfo, error) {
 	return ps.findChan(ctx, func(ci *ChannelInfo) bool {
 		if ci.Direction != DirOutbound {
 			return false
 		}
 		if ci.Settling {
 			return false
 		}
+
+		if ci.Channel != nil {
```

Comment:
Is this just an unrelated bug fix?

Reply:
Yes, this would break your paychmgr after the channel got settled on-chain